tests/pybboxes/annotations/test_annotations_conversion.py

```diff
@@ -0,0 +1,120 @@
+import pytest
+from pybboxes.annotations import Annotations
+import os
+import shutil
+import glob
+from collections import Counter
+from pycocotools.coco import COCO
+
+def count_files(directory, extensions):
+    all_files = []
+    for ext in extensions:
+        all_files.extend(glob.glob(f"{directory}/*{ext}"))
+    return Counter(file.split('.')[-1] for file in all_files)
+
+sample_yolo_dataset_path = str(os.path.join('tests', 'pybboxes', 'annotations', 'testing_data_yolo'))
```

Comment:
Can we upload these fixtures to a Huggingface ? and the test can run on the fly without the local dependency of the dataset (if exists, use, else, download) (huggingface python sdk already does this caching thing for you, then we will need only path-names or urls here.) I think it would be a nicer way of adding these kind of heavy fixtures.